setup.log

```diff
@@ -0,0 +1,356 @@
+2020-06-19 14:28:54,829 - PyBaMM setup - INFO - Starting PyBaMM setup
```

Comment:
Remove this file from the commit.  Don't know how it got through gitignore

Reply:
no idea why it appears. I found a new file `squared.py` appears when running the tests. Is it a bug?

Reply:
Yeah that is a bug, it's from the examples but should be getting removed. The setup.log file is from installation. You can safely delete it